### PR TITLE
fix: generator engine versioning

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

File: src/packages/sdk/src/getAllVersions.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import fetch from 'node-fetch'
+import fs from 'fs'
+import path from 'path'
+export { engineVersions } from './versions'
+
+async function getAllVersions() {
+  const url = `https://registry.npmjs.org/@prisma/cli`
+  const pkg = await fetch(url).then(r => r.json())
```
Comment (Jolg42) [inline src/packages/sdk/src/getAllVersions.ts:8]: We should use `getProxyAgent` here I think

**Review by Jolg42 — APPROVED** [review]
Nice job!

File: src/packages/sdk/src/getAllVersions.ts (hunk 1 above)
Comment (timsuchanek) [reply src/packages/sdk/src/getAllVersions.ts:8]: Not necessary, as this code is not being executed on any user machine.

Comment (Jolg42) [reply src/packages/sdk/src/getAllVersions.ts:8]: Indeed 👍

**Comment by geosigno** [comment]
I am still having the issue with the 2.9.0 version.

`TypeError: Cannot read property 'type' of undefined`

**Comment by hyochan** [comment]
Happening in `2.19.0` either. `2.18.0` and `2.20.0` is fine.

**Comment by janpio** [comment]
Please open an issue about this @hyochan and we can look into this. Thanks.

**Pull request merged**